bucket/sdl2_image.json

```
@@ -0,0 +1,123 @@
+            "$null, $currpath = strip_path (env 'PKG_CONFIG_PATH' $global) $pc_dir",
+            "write-host \"Adding $app to PKG_CONFIG_PATH.\"",
+            "env 'PKG_CONFIG_PATH' $global \"$pc_dir;$currpath\"",
+            "# 2.3.1.2 For cmake user",
+            "$null, $currpath = strip_path (env 'CMAKE_PREFIX_PATH' $global) $cm_dir",
+            "write-host \"Adding $app to CMAKE_PREFIX_PATH.\"",
+            "env 'CMAKE_PREFIX_PATH' $global \"$cm_dir;$currpath\"",
+            "# 2.3.2 Do it for the this sessions",
+            "# 2.3.2.1 For pck-config user",
+            "$null, $env:PKG_CONFIG_PATH = strip_path $env:PKG_CONFIG_PATH $pc_dir",
+            "$env:PKG_CONFIG_PATH = \"$pc_dir;$env:PKG_CONFIG_PATH\"",
+            "# 2.3.2.2 For cmake user",
+            "$null, $env:CMAKE_PREFIX_PATH = strip_path $env:CMAKE_PREFIX_PATH $cm_dir",
+            "$env:CMAKE_PREFIX_PATH = \"$cm_dir;$env:CMAKE_PREFIX_PATH\"",
+            "#",
+            "#",
+            "#-------------------------------------------------------------------#",
+            "# 3) Final cleanup",
+            "#-------------------------------------------------------------------#",
+            "# 3.1 Remove the original source folder",
+            "rm -r \"$src_dir\"",
+            ""
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "#-------------------------------------------------------------------#",
+            "# 1) Remove library link in the env variables",
+            "#-------------------------------------------------------------------#",
+            "# 1.1 Create helper variables.",
+            "$current_dir = \"$(current_dir $dir)\"",
+            "$pc_dir = \"$current_dir\\lib\\pkgconfig\"",
+            "$cm_dir = \"$current_dir\"",
+            "#",
+            "# 1.2 Delete env variables for future sessions",
+            "$was_in_path, $newpath = strip_path (env 'PKG_CONFIG_PATH' $global) $pc_dir",
+            "if($was_in_path) {",
+            "    write-host \"Removing $(friendly_path $pc_dir) from your path.\"",
+            "    env 'PKG_CONFIG_PATH' $global $newpath",
+            "}",
+            "$was_in_path, $newpath = strip_path (env 'CMAKE_PREFIX_PATH' $global) $cm_dir",
+            "if($was_in_path) {",
+            "    write-host \"Removing $(friendly_path $cm_dir) from your path.\"",
+            "    env 'CMAKE_PREFIX_PATH' $global $newpath",
+            "}",
+            "#",
+            "# 1.2 Delete env variables for this session",
+            "$was_in_path, $newpath = strip_path $env:PKG_CONFIG_PATH $pc_dir",
+            "if($was_in_path) { $env:PKG_CONFIG_PATH = $newpath }",
+            "$was_in_path, $newpath = strip_path $env:CMAKE_PREFIX_PATH $cm_dir",
+            "if($was_in_path) { $env:CMAKE_PREFIX_PATH = $newpath }"
+        ]
+    },
+    "checkver": {
+        "url": "https://www.libsdl.org/projects/SDL_image/",
+        "regex": "SDL2_image-([\\d.]+)\\.zip"
+    }
```

```suggestion
    },
    "autoupdate": {
        "url": [
            "https://www.libsdl.org/projects/SDL_image/release/SDL2_image-devel-$version-VC.zip",
            "https://www.libsdl.org/projects/SDL_image/release/SDL2_image-$version.zip"
        ]
    }
```
This should work after https://github.com/ScoopInstaller/Scoop/pull/3518 is merged into master branch.

---

Thanks for your suggestion, I will add it.